### PR TITLE
Add CRAN installation to README

### PR DESCRIPTION
**Pull request by batpigandme**
* Add instructions for installing from CRAN to README, since lobstr is now released.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,26 +66,26 @@ digging into the underlying \_\_ref\_\_erences:
...
+#> └─self = [1:0x7fdb60839ea8]
 ```
 
 A related tool is `obj_size()`, which computes the size of an object
 taking these shared references into account:
 
 ``` r
 obj_size(x)
-#> 680 B
+#> 4,000,048 B
```
Comment (batpigandme) [inline README.md]: @hadley I did this with a fresh install of lobstr from CRAN. I'm totally unclear as to why there's like a 6,000% increase in object size. 😞

Comment (hadley) [reply README.md]: What version of R do you have?

Comment (batpigandme) [reply README.md]: 3.5.1

Comment (hadley) [reply README.md]: Damn, I see it too. I better do a patch release 😦

Comment (hadley) [reply README.md]: Argh no, it goes away if I install from CRAN with `type = "source"`

Comment (batpigandme) [reply README.md]: OK, got it that's super weird.

Comment (hadley) [reply README.md]: Ok, figured it out with Jim's help — I'll need to do a patch release.

**Pull request merged**